src/mbgl/renderer/layers/render_line_layer.cpp

```diff
@@ -146,6 +144,12 @@ void RenderLineLayer::render(PaintParameters& parameters, RenderSource*) {
 
             optional<ImagePosition> posA = geometryTile.getPattern(linePatternValue.from);
             optional<ImagePosition> posB = geometryTile.getPattern(linePatternValue.to);
```

Comment:
I just pushed a fix for this issue – I think I didn't notice it at first because I quickly tested by zooming in from 0 so I didn't trigger the flashing.